features/FEATURE_BLE/targets/TARGET_CORDIO/doc/PortingGuide.md

```diff
@@ -144,7 +128,7 @@ public:
 
     virtual void start_reset_sequence();
 
-    virtual void handle_reset_sequence(uint8_t *msg);
+    virtual void handle_reset_sequence(uint8_t \*msg);
```

Comment:
could you remove the `\`

Reply:
Will do, but it's making the markdown go funny since it's processing as an unpaired asterisk.

Reply:
This should be in a code snippet or in ```, which will mean it won't render funny

Reply:
Ah, that explains it. Hello, meet my early work...